src/components/Table.jsx

```diff
@@ -273,7 +273,9 @@ export default function Table({ search, fetchBulkStyleSearch }) {
     const body = {
       filters: {
         ...(search && {
-          globalSearch: search?.replaceAll(" ", "")?.split(",")
+          globalSearch: search.includes(",")
```

Comment:
same code is repeated everywhere. Keep it in a single method and reuse it.

Reply:
addressed